plugins/rapid7_intsights/plugin.spec.yaml

```diff
@@ -615,3 +615,114 @@ actions:
+        type: '[]string'
+        required: false
+        example: ["Low"]
+      source_type:
+        title: Source Type
+        description: 'List of alerts source type. Allowed values: Application Store, Cyber Security Blog, Hacking News, Cyber Crime Forum, Hacktivism Forum, Social Media, Facebook, Twitter, LinkedIn, Google Plus, VK, Vimeo, YouTube, IRC Channel, IOC Block List, Credit Card Black Market, Paste Site, Data Leakage Website, Leaked Database, File Sharing Website, Gray Hat Website, Black Market, WHOIS servers, Company Website, Wikileaks, Pinterest, Tumblr, Instagram, Telegram, Webmail, Malware Analysis, Firehol, VRA, Other'
+        type: '[]string'
+        required: false
+        example: ["Application Store"]
+      network_type:
+        title: Network Type
+        description: 'List of network type. Allowed values: ClearWeb, DarkWeb'
+        type: '[]string'
+        required: false
+        example: ["DarkWeb"]
+      matched_asset_value:
+        title: Matched Asset Value
+        description: List of matched asset values
+        type: '[]string'
+        required: false
+        example: ["example.com"]
+      remediation_status:
+        title: Remediation Status
+        description: 'List of remediation statuses. Allowed values: InProgress, Pending, CancellationInProgress, Cancelled, CompletedSuccessfully, Failed'
+        type: '[]string'
+        required: false
+        example: ["InProgress", "Pending"]
+      source_date_from:
+        title: Source Date From
+        description: Start date to fetch from in Unix Millisecond Timestamp
+        type: string
+        required: false
+        example: 1633047083142
+      source_date_to:
+        title: Source Date To
+        description: End date to fetch to in Unix Millisecond Timestamp
+        type: string
+        required: false
+        example: 1633047102456
+      found_date_from:
+        title: Found Date From
+        description: Start date to fetch from in Unix Millisecond Timestamp
+        type: string
+        required: false
+        example: 0
+      found_date_to:
+        title: Found Date To
+        description: End date to fetch to in Unix Millisecond Timestamp
+        type: string
+        required: false
+        example: 1633047102456
+      assigned:
+        title: Alert Assignment
+        description: Show assigned/unassigned alerts
+        type: string
+        required: false
+        example: Assigned
+        enum:
+          - Assigned
+          - Unassigned
+      is_flagged:
+        title: Alert Flag Status
+        description: Show flagged/unflagged alerts
+        type: string
+        required: false
+        example: Flagged
+        enum:
+          - Flagged
+          - Unflagged
+      is_closed:
+        title: Closed Status
+        description: Status of the alert, either closed or open
+        type: string
+        required: false
```

Comment:
Please run a test with an enum value not set.

Reply:
I run test without any inputs:
```
{"body": {"meta": {}, "output": {"alert_ids": ["4142j8p1m2l8h77088n5eb0h"]}
{"body": {"meta": {}, "output": {"alert_ids": ["4142j8p1m2l8h77088n5eb1h"]}, "log": ""}, "type": "trigger_event", "version": "v1"}
```

Reply:
@r7-mgorny while this works in local testing, this will fail in-product if a user omits this input (even though it is an optional input, it treats the "empty" input as an empty string). You will need to add an empty string enum option to this input and any other enum inputs within the plugin.

Reply:
@mrinehart-r7 
Maybe we should add "any" or "both" enum instead of empty string in plugin.spec and then treat it as empty?